particles/Native/Wasm/source/TestParticle.kt

```diff
@@ -90,21 +107,21 @@ class TestParticle : Particle() {
         registerHandle("info", info)
 
         eventHandler("add") {
-          val newData = data.get()!!
-          newData.num = newData.num?.let { it + 2 } ?: 0.0
-          newData.txt = (newData.txt ?: "") + "!!!!!!"
-          this.data.set(newData)
+            val newData = data.get()!!
```

Comment:
Prefer `requireNotNull` to `!!`

Reply:
I switched it to use the elvis operator as this is more inline with TTT.